### ISSUE TITLE
Web SEO Title default 'Build with Flutterflow' not changing

### ISSUE THREAD

**Issue by jimbo43211** [FlutterFlow/flutterflow-issues#1981]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Web 'Site Description' default 'Build with Flutterflow' not changing despite own description being added.

### Expected Behavior

To be able to ammend the above and remove built with flitterlfow 

### Steps to Reproduce

Change Site Description field under Web Publishing tab.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

I am not sure where the code for this is 

### Context

As above, web app deployment, site description only showing as built with Flutterflow on Google, even when changed 

### Visual documentation

<img width="560" alt="Screenshot 2023-12-07 at 19 01 26" src="https://github.com/FlutterFlow/flutterflow-issues/assets/123547507/26852908-0356-40a0-9d3a-ad4fecee4664">

### Environment

```markdown
- FlutterFlow version: v4.0
- Platform: Mac
- Browser name and version: Mac
- Operating system and version affected: Mac
```

**Comment by jimbo43211** [comment]
This is also causing significant delay before I can fully advertise so would be grateful for help. I have tried to remove search engine indexing and navigate via my landing page in Wix but it wont remove from Google results.

**Comment by aemelyanovff** [comment]
@jimbo43211 Thanks for reporting the issue.

SEO tags for https://gpninja.flutterflow.app/ are set correctly:
![Screenshot 2023-12-15 at 1 34 06 PM](https://github.com/FlutterFlow/flutterflow-issues/assets/92294640/bb52744c-bb98-466f-b11f-d9422050c56e)

Why do you think it's not working?

**Comment by jimbo43211** [comment]
Could it be because I am using a custom domain. Are the SEO tags also applied to gpninja.uk?

**Comment by aemelyanovff** [comment]
I see the same SEO tags for gpninja.uk.

I see you "tried to remove search engine indexing" - if you uncheck "Allow Search Engine Indexing" then search engines won't re-index your web app. And note that changing this value will take effect only after you publish your project.

I'm closing the issue as not a bug. If you require further assistance, please reach out to FlutterFlow support.

**Comment by jimbo43211** [comment]
Hi, I have published it with enabling indexing with my own SEO title but still no luck...any idea why this could be?

**Comment by jimbo43211** [comment]
Hi, I have published it with enabling indexing with my own SEO title but still no luck...any idea why this could be and how to fix it?

James

**Comment by jimbo43211** [comment]
HiJust FYI I added a custom header and now it works. No idea why just thought to try!Sent from my iPhoneOn 18 Dec 2023, at 15:06, Anton Emelyanov ***@***.***> wrote:﻿

Hi, I have published it with enabling indexing with my own SEO title but still no luck...any idea why this could be?

I don't know, unfortunately. But I'm rather certain it's not a problem on the FlutterFlow side. Consider asking advice on https://community.flutterflow.io/.
